install/install_for_macos_linux.sh

```diff
@@ -44,12 +44,19 @@ curl -fSL $TIPI_URL --output ~/tipi.zip || wget -q $TIPI_URL -O ~/tipi.zip || ab
 info "Installing tipi in $INSTALL_FOLDER/bin"
 sudo unzip ~/tipi.zip -d $INSTALL_FOLDER -x LICENSE && rm ~/tipi.zip
 
+
 if [ $? -eq 0 ]; then
+    tipi_full_path=$INSTALL_FOLDER/bin/tipi
+    sudo chown ${USER:=$(/usr/bin/id -run)}:$USER $tipi_full_path
+    sudo chmod a+x,u+w $tipi_full_path
+
     info "tipi successfully installed. Installing the dependencies..."
-    sudo chmod +x $INSTALL_FOLDER/bin/tipi
-    $INSTALL_FOLDER/bin/tipi --help --dont-upgrade
+    
+    $INSTALL_FOLDER/bin/tipi --dont-upgrade run echo "Done"
```

Comment:
This I did so we don't spit out a huge blob of "--help" amidst the installation. Nice side effect: tipi will run the echo Done only after shipping all the tools